src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -600,6 +600,14 @@ private SmugglerProgressBase.Counts ProcessDocuments(SmugglerResult result, Buil
                     item.Document.Flags = item.Document.Flags.Strip(DocumentFlags.FromClusterTransaction);
                     item.Document.NonPersistentFlags |= NonPersistentDocumentFlags.FromSmuggler;
 
+                    if (_destination is DatabaseDestination
+                        && _options.SkipRevisionCreation
+                        && _options.OperateOnTypes.HasFlag(DatabaseItemType.RevisionDocuments)
```

Comment:
I'm wondering if this is necessary (I'm aware that we've agreed to add it). Maybe if someone does not want to create revisions then we should just add the 'SkipRevisionCreation' without checking for this? @ayende what is your opinion?

Reply:
Yes, we can just always add that.
For that matter, since this is non persistent flag, we don't even care if this goes to a db destination.
Stream destination doesn't care.

Reply:
removed all conditions except `_options.SkipRevisionCreation`